learning/katas/java/.idea/study_project.xml

```diff
@@ -0,0 +1,3151 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this related?

Reply:
No. Ignore that file - I will remove it from the PR.